File: backend/src/main/java/com/ftn/activityapp/dto/freetime/FreeTimeActivityDto.java
package com.ftn.activityapp.dto.freetime;

import com.ftn.activityapp.enums.FreeTimeCategory;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class FreeTimeActivityDto {
    private String title;
    private String description;
    private FreeTimeCategory category;
    private Integer durationMinutes;
    private String intensity;
    private String imageKey;
    private Integer sortOrder;
}

File: mobile-app/app/src/main/java/com/example/activityapp/data/remote/dto/freetime/FreeTimeActivityDto.kt
package com.example.activityapp.data.remote.dto.freetime

data class FreeTimeActivityDto(
    val title: String,
    val description: String,
    val category: String,
    val durationMinutes: Int,
    val intensity: String,
    val imageKey: String,
    val sortOrder: Int
)